content/architecture/infrastructure/deployment/altinn-apps/_index.md

```diff
@@ -18,92 +18,139 @@ The following diagram shows the deployment architecture for Altinn Studio togeth
 
 [See fullscreen] or [download as visio]
 
-### Kubernetes
-Each ["ServiceApp"]({{< ref "/about/wordlist" >}}) created in Altinn Studio is deployed
-to Altinn Studio Apps as separate applications running in Docker Containers.
+## Kubernetes
+Each ["App"]({{< ref "/about/wordlist" >}}) created in Altinn Studio is deployed
+to Altinn Apps as separate applications running in Docker Containers.
 The containers will be orchestrated by Kubernetes.
 
-#### Deployment
-The Altinn Studio Service Apps are deployed as sets of Docker Containers defined as Kubernetes Deployment.
-The deployment will be in the service owners own Kubernetes Cluster.
+### Deployment
+The Altinn App is deployed as sets of Docker Containers defined as Kubernetes Deployment.
+The deployment will be in the organisation's own Kubernetes Cluster.
 
-> All service owners will have their own Kubernetes Cluster.
+> All organisations will have their own Kubernetes Cluster.
 
-Each set of Kubernetes Deployment will be scaled based on the usage of the service.
+Each set of Kubernetes Deployment will be scaled based on the usage of the app.
 Scaling of the Deployment will be configurable in the Kubernetes Deployment and automatically scaled using
 [Kubernetes Replicaset](https://kubernetes.io/docs/reference/glossary/?all=true#term-replica-set).
 
-> Kubernetes Clusters and Serviceapps can do horizontal autoscaling.
+> Kubernetes Clusters and Altinn Apps can do horizontal autoscaling.
 > It is currently not decided if and how autoscaling will be used.
 
-The service container will consist of the runtime application and service specific code and configuration.
+The Altinn App in Kubernetes will be configured as a Kubernetes Service which consist of the runtime application
+and the Altinn App with related code and configuration.
```

Comment:
Should it be pointed out that for MVP the app will consist of the latest runtime application and the Altinn App will be related code and configuration from the master branch of the repository?

Reply:
Done